api/OfflineAudioContext.json

```diff
@@ -292,7 +292,7 @@
               "version_added": "6"
             },
             "safari_ios": {
-              "version_added": null
+              "version_added": "6"
```

Comment:
I tried https://mdn-bcd-collector.appspot.com/tests/api/webkitOfflineAudioContext in iOS 6 (BrowserStack simulator) and it looks like it's not supported, while https://mdn-bcd-collector.appspot.com/tests/api/webkitAudioContext is.

But it is in iOS 7 on a real device.

It seems like maybe `OfflineAudioContext` came a release later, but this is consistent with the existing data, and the difference is minor, so I'm going to look away and say it's fine.